e2e/vue3-vite-e2e/tests/docs.spec.ts

```diff
@@ -0,0 +1,45 @@
+import {
+  checkFilesExist,
+  readFile,
+  runNxCommandAsync,
+  uniq,
+} from '@nrwl/nx-plugin/testing';
+
+jest.setTimeout(60000);
+
+describe('docs e2e', () => {
```

Comment:
Skipping this test for now, the doc generator will need to be updated at some point. I think the current version breaks under Vite 4.